leaderboard/service/get_leaders_test.go

```diff
@@ -0,0 +1,116 @@
+package service_test
+
+import (
+	"context"
+	"fmt"
+	"math"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/topfreegames/podium/leaderboard/database"
+	"github.com/topfreegames/podium/leaderboard/model"
+	"github.com/topfreegames/podium/leaderboard/service"
+)
+
+var _ = Describe("Service GetLeaders", func() {
+	var ctrl *gomock.Controller
+	var mock *database.MockDatabase
+	var svc *service.Service
+
+	var leaderboard string = "leaderboardTest"
+	var totalMembers int = 10
+	var pageSize int = 3
+	var page int = 1
+	var order string = "asc"
+
+	var start int = (page - 1) * pageSize
+	var stop int = start + pageSize - 1
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mock = database.NewMockDatabase(ctrl)
+
+		svc = &service.Service{mock}
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	It("Should return members slice if all is OK", func() {
+		membersDatabaseReturn := []*database.Member{
+			&database.Member{
+				Member: "member1",
+				Score:  float64(1),
+			},
+			&database.Member{
+				Member: "member2",
+				Score:  float64(2),
+			},
+			&database.Member{
+				Member: "member3",
+				Score:  float64(3),
+			},
+		}
+
+		membersReturn := []*model.Member{
+			&model.Member{
+				PublicID: "member1",
+				Score:    1,
+				Rank:     0 + 0 + 1,
```

Comment:
I didn't understand why you are representing the Rank as a sum of three numbers instead of the sum result directly.